sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpClient.cs

```diff
@@ -99,7 +99,7 @@ internal class AmqpClient : TransportClient
                 ServiceEndpoint,
                 credential,
                 options.TransportType,
-                options.Proxy);
+                options.WebProxy);
```

Comment:
I'm not sure where this came from, but I question the drift from the naming for the same concept in Event Hubs across the languages.

Reply:
This was based on a comment from Clemens in the Api View.